server/routes/private.js

```diff
@@ -110,3 +110,52 @@ router.get('/test', async (req, res) => {
 
 module.exports = router
 
+router.post('/api/add-recurring-schedule', async (req, res) => {
+    // Validate appropriate parameters are passed to view workers at each school 
+    const paramSchema = Joi.object({
+        schoolId: Joi.number().integer().required(), //The student must specify their school ID in order to view the workers
+        schoolName: Joi.string().max(30), // The default value is available which is handled by the database Create Table logic 
```

Comment:
Hi Simran,
I think the endpoint ending should be updated to be more clear as to what this endpoint is doing - perhaps `get-workers-for-school` or something along that line. Also, you can remove schoolName as per the comments above as well.

Reply:
Thank you Amy! The changes have been made as per your comments!